src/frontend/komponenter/Fagsak/useFagsakApi.ts

```diff
@@ -160,7 +160,7 @@ const useFagsakApi = (
         ) {
             if (skjemaetHarEndringer) {
                 settSenderInn(true);
-                apiOpprettBeregning(fagsak.id, {
+                apiOpprettBeregning(fagsak, {
```

Comment:
I prefer to pass the id so that the functions are "dum"/simple.

Reply:
I think the caller may be more complex when the functions become simpler. The profit/cost depends on how many times the function will be used. For now, If we dont need special error handling I prefer passing the object so that the caller may not need to change even we change the endpoints/parameters in the future.
But it is not a big issue anyway, its also ok for me to use IDs.

Reply:
The only thing I am worried about is if we suddenly have different logic several places to retrieve aktiv (vedtak, behandling) and so forth. That way some nasty bugs would be hard to uncover.